packages/react-reconciler/src/__tests__/ReactFlushSync-test.js

```diff
@@ -72,7 +117,7 @@ describe('ReactFlushSync', () => {
         await waitForPaint(['1, 1']);
       }
     });
-    expect(root).toMatchRenderedOutput('1, 1');
+    expect(getVisibleChildren(container)).toEqual('1, 1');
```

Comment:
What do you think about creating a matcher for this? Seems common and we're starting to copy this help over to multiple places. It would be nice to standardize the ReactDOM tests asserting container content on one patter because it's all over the place right now

Reply:
Yeah it's generally very useful however there are some special cases in ReactDOMFloat tests where we do even more special casing because we need to assert certain things like scripts which the basic version of this ignores. Maybe instead of a matcher we just have a test util for this function so it's a little clearer what's happening when you choose to specialize the implementation. Or maybe the float one can be made the default one